rcmgr.go

```diff
@@ -423,13 +424,93 @@ func newStreamScope(dir network.Direction, limit Limit, peer *peerScope, rcmgr *
 	return &streamScope{
 		resourceScope: newResourceScope(limit,
 			[]*resourceScope{peer.resourceScope, rcmgr.transient.resourceScope, rcmgr.system.resourceScope},
-			fmt.Sprintf("stream-%d", rcmgr.nextStreamId()), rcmgr.trace, rcmgr.metrics),
+			streamScopeName(rcmgr.nextStreamId()), rcmgr.trace, rcmgr.metrics),
 		dir:   dir,
 		rcmgr: peer.rcmgr,
 		peer:  peer,
 	}
 }
 
+func IsSystemScope(name string) bool {
+	return strings.HasPrefix(name, "system")
```

Comment:
This would return `true` for a span in the system scope. Is that intended?

Reply:
Not indended, will fix